notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -2661,6 +2661,54 @@ monster [The fire demon Surtur.]
 magic portal [Doctor Strange appeared to Thor through a portal that the doctor had conjured up.]
 
 
+movie: Beyond Skyline (2017)
+============================
+
+:: Title
+Beyond Skyline
+
+:: Description
+It is up to a ragtag band of survivors of an initial alien attack on the Earth
+to stop the aliens from completing their invasion. The film is a sequel to the
+2010 film Skyline, set concurrently with the events of the first film.
+
+:: Date
+2017-10-14
+
+:: References
+https://en.wikipedia.org/wiki/Beyond_Skyline
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+attack from outer space [The story follows a small group of people who become caught up in an alien invasion of Earth.]
+
+:: Major Themes
+father and son [The hard-boiled detective Mark Corley risked life and limb to save his son, Trent, who'd
+been taken captive by invading aliens aboard their mothership.]
+extraterrestrial being [Alien invaders were going around abducting and/or killing people.]
+alien abduction [The invading aliens sucked any number of Los Angeles residents into their various spaceships by means of an ethereal blue light. In particular, Mark and his son Trent were taken aboard the alien mothership whereupon Trent has his brain removed and placed in the body of an alien bio-mechanical creature.]
+giant monster [Giant alien bio-mechanical creatures were rampaging around. The film culminated with a big fight between two of the giant creatures, one controlled by the alien leader, the other by Trent.]
```

Comment:
Just a reflection: there are natural giant monsters like Godzilla, and artificial giant monsters that blur the line between robot and creature. A distinction worth making?

Reply:
I see where you're coming from, but lump them all together for now?